formal/promela/src/testbuilder.py

```diff
@@ -50,6 +50,17 @@ def clean(model):
         os.remove(file)
 
 
+def archive(model):
+    print(f"Archiving spin files for {model}")
+    files = glob.glob(model + '*.trail')
+    files += glob.glob(model + '*.spn')
+    date = datetime.now().strftime("%Y%m%d-%H%M%S")
```

Comment:
I was under the impression only files not moved elsewhere elsewhere were intended for archiving.

Does it make sense to archive all files within the folder in that case? It would likely make it easier to see the effects of changes to the model.